docs/status-code-comparison-to-docker.md

```diff
@@ -0,0 +1,597 @@
+<!--[metadata]>
++++
+title = "API response codes"
+description = "Swarm API response codes"
+keywords = ["docker, swarm, response, code,  api"]
+[menu.main]
+parent="workw_swarm"
+weight=99
++++
+<![end-metadata]-->
+
+This is a documentation describing the difference between Swarm and Docker Engine in the aspect of  api response status codes.
+
+# Understand the Swarm vs. Engine response codes
+
+Four methods are included, and they are GET, POST, PUT and DELETE.
+
+The comparison is based on api v1.22, and all Docker Status Codes in api v1.22 are referenced from [docker-remote-api-v1.22](https://github.com/docker/docker/blob/master/docs/reference/api/docker_remote_api_v1.22.md
+).
+
+#GET
```

Comment:
These should be second level heads and need a space between the `##` and the first letter.

#GET > ## GET
#POST > ## Post